domain/src/test/java/org/oppia/domain/oppialogger/analytics/AnalyticsControllerTest.kt

```diff
@@ -97,7 +101,7 @@ class AnalyticsControllerTest {
     analyticsController.logTransitionEvent(
```

Comment:
Will it not make more sense if we logged the event using the `oppiaLogger` itself ? I'm saying because in the real scenario `oppiaLogger` will be used to log events and `analyticsController` will be used through it. Here and elsewhere in this file.

Reply:
The way I saw it is that the actual logging functionality is implemented here. Oppia Logger is only routing the logging to this so it made more sense to keep the test here. Also, another reason is that the OppiaLoggerTest would get extremely large if we take these nuances into that test. I intend to add the OppiaLoggerTests to only check if its calling the right functions in the underlying loggers (by checking the state) . But the network/no network tests etc can stay in the underlying logger.